### PR TITLE
[doc] 6.8 deprecation logging bug - known issue

### PR DESCRIPTION
**Pull request by pgomulka**
known issue doc on a bug #60561

### CODE REVIEW

File: docs/reference/release-notes/6.7.asciidoc [hunk 1]
```diff
@@ -207,6 +207,14 @@ will remain as `STOPPING` until all policies that do not exist are removed from
 any indices by using the <<ilm-remove-policy,Remove Policy API>>.
 (issue: {es-issue}/40824[#40824])
 
+Deprecation Logging::
+
+In a mixed cluster environment where one of the nodes is 6.7 but coordinating node is 6.7
+an exception will be thrown when a deprecation warning is emitted on pre 6.7 node.
+it was introduced by (PR: {es-pull}/37622[#37622]) in order to speed up the performance of header warning parsing.
+
+(issue: {es-issue}/60561[#60561])
```
Comment (jrodewig) [inline docs/reference/release-notes/6.7.asciidoc:216]:
```suggestion
(issue: {es-issue}/60561[#60561])
```

File: docs/reference/release-notes/6.7.asciidoc [hunk 2]
```diff
@@ -207,6 +207,14 @@ will remain as `STOPPING` until all policies that do not exist are removed from
 any indices by using the <<ilm-remove-policy,Remove Policy API>>.
 (issue: {es-issue}/40824[#40824])
 
+Deprecation Logging::
+
+In a mixed cluster environment where one of the nodes is 6.7 but coordinating node is 6.7
+an exception will be thrown when a deprecation warning is emitted on pre 6.7 node.
+it was introduced by (PR: {es-pull}/37622[#37622]) in order to speed up the performance of header warning parsing.
```
Comment (jrodewig) [inline docs/reference/release-notes/6.7.asciidoc:214]: I left a suggestion for wording here. Please review to ensure I didn't introduce errors.
The original text isn't1 clear about which node is pre-6.7.

```suggestion
Deprecation logging::
If the coordinating node for the upgrade uses {es} 6.7 and another node uses an
earlier 6.x or 5.x version, {es} will emit a deprecation warning and return an error. This
behavior was introduced {es-pull}/37622[#37622] to speed up header warning
parsing.
```

**Review by jrodewig — APPROVED** [review]
Looks good overall. I left a comment regarding clarity.

File: docs/reference/release-notes/6.7.asciidoc (hunk 2 above)
Comment (pgomulka) [reply docs/reference/release-notes/6.7.asciidoc:214]: good point, it did not made sense :D thanks

**Pull request merged**